doc/guides/security_release_process.md

```diff
@@ -2,118 +2,97 @@
-  Make sure to update the date in the slug so that it will move to
-  the top of the blog list, and not that it updates the
-  banner on Node.js org to indicate the security release(s) are
-  available. ***LINK TO PR***
-
-  *Note*: If the release blog obviously points to the people having caused the
-  issue (for example when explicitly mentioning reverting a commit), adding
-  those people as a CC on the PR for the blog post to give them a heads up
-  might be appropriate.
-
-* [ ] Email foundation contact to tweet out nodejs-sec announcement from
-  foundation twitter account. FIXME - who is this contact?
-
-* [ ] Create a PR to update the Node.js version in the official docker images.
-  * Checkout the docker-node repo.
-  * Run the update.sh using the `-s` option so that ONLY the Node.js
-    versions are updated. At the request from docker (and because
-    it is good practice) we limit the changes to those necessary in
-    security updates.
-  * Open a PR and get volunteer lined up earlier to approve.
-  * Merge the PR with the merge button.
-  * Checkout the [official-images](https://github.com/docker-library/official-images)
-    repository .
-  * In the docker-node repository run the
-    [generate-stackbrew-library.sh]( https://github.com/nodejs/docker-node/blob/master/generate-stackbrew-library.sh)
-    script and replace official-images/library/node with the output generated.
-    ```console
-    $ ./generate-stackbrew-library.sh > .../official-images/library/node
-    ```
-  * Open a PR with the changes to official-images/library/node making sure to
-    @mention the official images.
-    [maintainers](https://github.com/docker-library/official-images/blob/master/MAINTAINERS).
-    In addition, make sure to prefix the PR title with `[security]`.
-  * Send an email to the
-    [maintainers](https://github.com/docker-library/official-images/blob/master/MAINTAINERS)
-    indicating that the PR is open.
-
-* [ ] If we allocated CVES:
-  * [ ] Ensure that the announced CVEs are reported to Mitre as per the
-    [cve_management_process](https://github.com/nodejs/security-wg/blob/master/processes/cve_management_process.md).
-  * [ ] Ensure that the announced CVEs are updated in the cve-management
-    repository as per the
-    [cve_management_process](https://github.com/nodejs/security-wg/blob/master/processes/cve_management_process.md)
-    so that they are listed under Announced.
+* [ ] [Unlock CI](https://github.com/nodejs/build/blob/master/doc/jenkins-guide.md#after-the-release)
+> FIXME: Whoever is doing the Node.js release should have sufficient privs to
+> do this
+
+* [ ] Post-release announcement in reply [email][]: ***LINK TO EMAIL***
+
+* [ ] Post-release announcement to Nodejs.org blog: ***LINK TO BLOG POST***
+  * (Re-PR the pre-approved branch from nodejs-private/nodejs.org-private to
+    nodejs/nodejs.org)
+
+* [ ] Email foundation [contact](FIXME) to tweet an announcement
```

Comment:
Didn't do this in the last release, and I'm not sure who should, or who this contact is. Anyone know? I'm OK with deleting it . @RRomoff , from https://github.com/nodejs/admin/issues/454, perhaps you know?